app.py

```diff
@@ -17,19 +18,19 @@
 # Zluda hijack
 import rvc.lib.zluda
 
+# Run prerequisites
```

Comment:
not a big fan of this import rework, makes some no sense changes

Reply:
Then, I think we can keep only autoflake. It will automatically remove imports that are not used in the code.

I will update the PR with it now.